### ISSUE TITLE
用語集：サイズの大きな (テキスト) の見直し

### ISSUE THREAD

**Issue by momdo** [waic/wcag21#1606]
「サイズの大きな (テキスト) 」について、見直してみました。

https://www.w3.org/TR/2018/REC-WCAG21-20180605/#dfn-labels
https://waic.jp/docs/WCAG21/#dfn-large-scale

以下の順に並べています。
- 原文
- 現行訳
- 提案訳

---

サイズの大きな (テキスト) (large scale (text))

> with at least 18 point or 14 point bold or font size that would yield equivalent size for Chinese, Japanese and Korean (CJK) fonts

現行：少なくとも 18 ポイント、又は 14 ポイントの太字。あるいは、中国語、日本語、及び韓国語 (CJK) のフォントは、それと同等の文字サイズ。

提案：少なくとも 18 ポイント、又は 14 ポイントの太字。又は、中国語、日本語、及び韓国語 (CJK) のフォントについては同等のサイズ。

NOTE
> Fonts with extraordinarily thin strokes or unusual features and characteristics that reduce the familiarity of their letter forms are harder to read, especially at lower contrast levels.

現行：特別に細い線のフォント、又は文字の形が分かりにくくなるような独特の見た目や特徴のあるフォントは、コントラストが低い場合に特に読みづらい。

提案：ストロークが極端に細いフォント、又は文字の形状に馴染みの少ない独特な特徴を持つフォントは、コントラストが低い場合に特に読みづらくなる。

NOTE
> Font size is the size when the content is delivered. It does not include resizing that may be done by a user.

現行：ここでいう文字サイズは、コンテンツが提供される際のサイズであり、利用者によるサイズ変更は含まれない。

提案：ここでいうフォントサイズは、コンテンツ配信時のサイズであり、利用者によるサイズ変更は含まれない。

NOTE
> The actual size of the character that a user sees is dependent both on the author-defined size and the user's display or user-agent settings. For many mainstream body text fonts, 14 and 18 point is roughly equivalent to 1.2 and 1.5 em or to 120% or 150% of the default size for body text (assuming that the body font is 100%), but authors would need to check this for the particular fonts in use. When fonts are defined in relative units, the actual point size is calculated by the user agent for display. The point size should be obtained from the user agent, or calculated based on font metrics as the user agent does, when evaluating this success criterion. Users who have low vision would be responsible for choosing appropriate settings.

現行：利用者が目にする文字の実際のサイズは、コンテンツ制作者が指定したサイズと利用者のディスプレイあるいはユーザエージェントの設定の両方に依存している。多くの主流となっている本文テキストで用いられるフォントにおいて、14 ポイントと 18 ポイントは、1.2em と 1.5em、又は、 (本文フォントが 100％であると仮定して) デフォルトサイズの 120%と 150%に、おおよそ同等である。しかし、制作者は、使用する特定のフォントについて、このことをチェックしておく必要がある。フォントが相対単位で定義されている時、実際に表示される文字サイズのポイント数は、ユーザエージェントによって計算される。この達成基準について評価する時には、文字サイズのポイント数は、ユーザエージェントから取得されるべきであり、又はユーザエージェントが行うフォントの計算基準に基づいて計算するべきである。ロービジョンの利用者については、自分で適切な設定を選択することを想定している。

提案：利用者が目にする文字の実際のサイズは、コンテンツ制作者が定義したサイズと、利用者のディスプレイ又はユーザエージェントの設定の両方に依存している。多くの主流となっている本文テキストのフォントでは、14 ポイントと 18 ポイントは、1.2 em と 1.5 em、又は、(本文フォントが 100％であると仮定して) 本文テキストのデフォルトサイズの 120％と 150％にほぼ相当する。しかし、コンテンツ制作者は使用する特定のフォントについてこれを確認する必要がある。フォントが相対単位で定義されている場合、実際に表示されるポイントのサイズは、ユーザエージェントによって算出される。ポイントのサイズは、この達成基準を評価するときに、ユーザエージェントから取得する、又はユーザエージェントが行うようにフォントメトリックに基づいて算出されるべきである。ロービジョンの利用者は、利用者自身で適切な設定を選択する責任がある。

NOTE
> When using text without specifying the font size, the smallest font size used on major browsers for unspecified text would be a reasonable size to assume for the font. If a level 1 heading is rendered in 14pt bold or higher on major browsers, then it would be reasonable to assume it is large text. Relative scaling can be calculated from the default sizes in a similar fashion.

現行：フォントサイズを指定せずにテキストを用いる際は、サイズを指定していないテキストに対して主要ブラウザで用いられる最小のフォントサイズをそのテキストのサイズとみなすのが妥当であろう。もし、レベル 1 の見出しが、主要なブラウザで 14 ポイントの太字あるいはそれ以上のサイズでレンダリングされるならば、それは「サイズの大きな」テキストであると考えてよい。相対的な拡大縮小は、同様の方法でそのデフォルトのサイズから算出することが可能である。

提案：フォントサイズを指定せずにテキストを使用する場合、主要なブラウザで不特定のテキストに使用される最小のフォントサイズが、フォントサイズとして妥当であろう。主要なブラウザで、レベル 1 の見出しが 14 ポイント以上の太字でレンダリングされる場合、それは「サイズの大きな」テキストと見なすのが妥当であろう。相対的な拡大縮小は、同様の方法でデフォルトサイズから算出可能である。

NOTE
> The 18 and 14 point sizes for roman texts are taken from the minimum size for large print (14pt) and the larger standard font size (18pt). For other fonts such as CJK languages, the "equivalent" sizes would be the minimum large print size used for those languages and the next larger standard large print size.

現行：半角の英数字のテキストにおける 18 ポイント及び 14 ポイントのサイズは、拡大印刷の最小サイズ (14 ポイント) と標準的な大きい文字サイズ (18 ポイント) に基づいている。例えば、CJK 言語のようなその他の文字については、「同等な」サイズはその言語での拡大印刷の最小サイズと拡大印刷でその次に大きな標準のサイズとなる。

提案：ローマン体テキストの 18 ポイント及び 14 ポイントのサイズは、大活字の最小サイズ (14 ポイント) とそれより大きな標準のフォントサイズ (18 ポイント) から選ばれた。CJK 言語のようなその他のフォントの場合、「同等の」サイズは、その言語で使用される最小の大活字のサイズと、その次に大きな、標準の大活字のサイズとなる。

**Comment by momdo** [comment]
Derived #98
